app/src/lib/git/core.ts

```diff
@@ -184,64 +182,66 @@ export async function git(
 
       throw err
     })
-  })
 
-  const exitCode = result.exitCode
+    const exitCode = result.exitCode
```

Comment:
All the code from this file below this line has just been indented. There is a very important (and long) comment in `trampoline-environment.ts` that explains why, but just a summary/spoiler: we want `withTrampolineEnv` to get any result returned by `git`/`spawnAndComplete`, and catch any error thrown by them.

That way, `withTrampolineEnv` can tell when the git operation succeeded or not and, according to that, store any SSH key passphrase associated to the git operation or not.